Index.html

```diff
@@ -35,19 +35,38 @@
 
     <main>
       <div class="carousel">
-        <div class="carousel-images">
+        <div class="carousel-item">
+          <div  class="carousel-text">
```

Comment:
If it's under div with name `carousel-item`, there should need to be `item `in `carousel-text` too